quirks/password-rules.json

```diff
@@ -35,6 +35,9 @@
     "brighthorizons.com": {
         "password-rules": "minlength: 8; maxlength: 16;"
     },
+    "capitalone.com": {
+        "password-rules": "minlength: 8; maxlength: 32; required: lower, upper; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-_./\@$*&!#];"
```

Comment:
Because lower, upper, and digit are required, you don’t need to specify them in allowed.